src/Tribe/Templates.php

```diff
@@ -543,8 +543,8 @@ public static function showInLoops( $query ) {
 		 **/
 		public static function getTemplateHierarchy( $template, $args = array() ) {
 			if ( ! is_array( $args ) ) {
+				$passed        = [ $template, $args ];
```

Comment:
This does not seem to be backwards compatible with the $backwards_map usage below where it still supports other arguments coming from func_get_args()

Reply:
```suggestion
				$passed        = func_get_args();
```

Reply:
True, I didn't consider that piece. Because we don't have any internal usage for that it seems. Do you feel like just moving this up would solve the issue?